expat/doc/reference.html

```diff
@@ -1112,12 +1113,40 @@ <h4 id="XML_Parse">XML_Parse</h4>
 <code>isFinal</code> parameter informs the parser that this is the last
 piece of the document. Frequently, the last piece is empty (i.e.
 <code>len</code> is zero.)
+</p>
+
+<p>
 If a parse error occurred, it returns <code>XML_STATUS_ERROR</code>.
 Otherwise it returns <code>XML_STATUS_OK</code> value.
 Note that regardless of the return value, there is no guarantee that all
 provided input has been parsed; only after <a href="#isFinal">the
 concluding call</a> will all handler callbacks and parsing errors have
 happened.
+</p>
+
+<p>
+Function <code>XML_Parse</code> is mostly a convenience wrapper pairing
+calls to <code><a href="#XML_ParseBuffer">XML_ParseBuffer</a></code>
+and <code><a href="#XML_GetBuffer">XML_GetBuffer</a></code>.
```

Comment:
Maybe turn it the other way around: "GetBuffer and ParseBuffer" instead of "ParseBuffer and GetBuffer".